xkb_utf32_to_keysym.h

```diff
@@ -0,0 +1,7 @@
+/* As of Feb 26, 2020, xkb_utf32_to_keysym.c exists only in development code
+ * for xkbcommon/libxkbcommon. This source code is to adapt
+ * https://github.com/xkbcommon/libxkbcommon/commit/0345aba082c83e9950f9dd8b7ea3bf91fe566a02
+ * which Jaroslaw Kubik authored and @bluetech committed.
+ * */
+
+static xkb_keysym_t xkb_utf32_to_keysym(uint32_t ucs);
```

Comment:
This file is not included anywhere and can just be removed.

Reply:
I just added this because the compiler told me to. If you're sure, I'll delete it.

Reply:
Huh? But... the file is not referenced anywhere...? I thought.

Reply:
Nah, you were right. I deleted it, and the compiler threw no warning.

I found so many errors at compile-time that I had forgotten what does what.